libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/PredictiveCacheController.kt

```diff
@@ -59,28 +58,21 @@ class PredictiveCacheController @JvmOverloads constructor(
 ) {
     private var map: MapboxMap? = null
 
-    private val onMapChangeListener = object : OnMapChangedListener {
-        override fun onMapChange(mapChange: MapChange) {
-            when (mapChange) {
-                MapChange.DID_FINISH_LOADING_STYLE -> {
-                    map?.let { map ->
-                        val tileStoreResult =
-                            TileStoreManager.getTileStore(map.getResourceOptions())
-                        createMapsControllers(tileStoreResult) { tileStore ->
-                            val currentMapSources = mutableListOf<String>()
-                            traverseMapSources(map) { id ->
-                                currentMapSources.add(id)
-                            }
-                            updateMapsControllers(
-                                currentMapSources,
-                                PredictiveCache.currentMapsPredictiveCacheControllers(),
-                                tileStore
-                            )
-                        }
+    private val onStyleLoadedListener = object : OnStyleLoadedListener {
+        override fun onStyleLoaded() {
```

Comment:
note that the OnStyleLoadedListener matches previous `DID_STYLE_FULLY_LOADED`.
and `MapChange.DID_FINISH_LOADING_STYLE` matches the new `OnStyleDataLoadedListener` when its associated `type` is `style`.

cc @Guardiola31337

Reply:
@pengdev 
Should be using `OnStyleDataLoadedListener` then? 🤔 

In any case, I think in this particular case is safer using `OnStyleLoadedListener`.

cc @d-prukop

Reply:
cc @kkaefer

Reply:
yeah, the `OnStyleLoadedListener` means the style, source and sprite are all loaded. It will be safer to use, but it will be triggered a little latter than the OnStyleDataLoadedListener when its associated type is style.